docs/guides/advise/network-policy.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: 'Using advise network-policy'
-weight: 10
+weight: 20
+description: >
+  Generate network policies based on captured output.
```

Comment:
Can we make it an exact match of https://github.com/kinvolk/inspektor-gadget/blob/main/cmd/kubectl-gadget/advise/network-policy.go#L40? Same for others.

Reply:
Alright, I've made all descriptions match the help tags. In three cases (the profile gadgets and snapshot socket), I modified the help tags to match the description I had written. I think it's good that these match, but it bothers me that there's so many places that we need to manually sync (there's also the README, which I also updated for my three changes). I wonder if there's a better way of doing that...